hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -197,6 +202,10 @@ export default function HackathonDetails(props: IProps) {
     </Modal>
   )
 
+  const closeEditModal = (isOpened: boolean) => {
+    setEditModalOpened(isOpened)
+  }
```

Comment:
the name of this function doesn't really match what it does, in fact, why even have this function at all when you just pass the value right into another function?